README.md

```diff
@@ -20,14 +20,14 @@ The purpose of the library is to provide a light, simple and general tracing sol
 ## Compromises
 
 * The traces are stored as ASCII arrays in the flash memory (pretty high memory consumption). Therefore, it is not necessary to:
- * encode/decode the trace messages on the fly (this may take too much CPU time) or 
- * have external dev-env dependencies to encode the traces compile time and an external application to decode the traces. 
+  * encode/decode the trace messages on the fly (this may take too much CPU time) or
+  * have external dev-env dependencies to encode the traces compile time and an external application to decode the traces.
```

Comment:
Were the above extra spaces intentional?

Reply:
yes - intended because related to previous topic..